tests/test_bot.py

```diff
@@ -363,6 +364,56 @@ def test_send_reply(self):
                                 attachments='null')
 
 
+class TestSendMessageWithMessageAttachments(TestPhialBot):
+    '''Test phial's send_message function with message attachments'''
+
+    def test_send_message(self):
+        self.bot.slack_client = MagicMock()
+        self.bot.slack_client.api_call = MagicMock(return_value="test")
+        message = Response(channel="channel_id",
+                           attachments=[MessageAttachment(
+                                fallback="fallback",
+                                author_name="John Doe",
+                                author_link="https://example.com/author",
+                                author_icon="https://example.com/author.jpg",
+                                color="#36a64f",
+                                title="Title",
+                                title_link="https://example.com",
+                                image_url="https://example.com/image.jpg",
+                                text="Go to Example Website",
+                                footer="Footer text",
+                                footer_icon="https://example.com/footer.jpg",
+                                thumb_url="https://example.com/thumb.jpg",
+                                fields=[
+                                    MessageAttachmentField(
+                                        title="Established",
+                                        value="2008",
+                                        short=False),
+                                    MessageAttachmentField(
+                                        title="Users",
+                                        value="27 Million",
+                                        short=True)])])
+        self.bot.send_message(message)
+
+        attachments = """[{"fallback": "fallback", "color": "#36a64f",
```

Comment:
Super nitpicky, but as could we spread the json over lines as if it were pretty printed e.g.
```python
expected_json = """
{
  "fallback": "fallback",
  "fields": {
    {
      "title": "users",
      etc...
    }
  },
  etc...
}
"""
```

Makes it easier to comprehend, as well as being more maintainable in the future if we have to add or change fields

Reply:
This is what I initially wanted to do, but the test is expecting it without the whitespace. As you can see from what I've already done I've had to go as far as removing the new line characters

Reply:
Could you not have it over mutiple lines then check that `json.loads(expected_json)` matches what we send through. Ths will remove any superfluous whitespace.